tests/Umbraco.Tests.Integration/ManagementApi/TemporaryFile/DeleteTemporaryFileControllerTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Linq.Expressions;
+using System.Net;
+using NUnit.Framework;
+using Umbraco.Cms.Api.Management.Controllers.TemporaryFile;
+using Umbraco.Cms.Core.Models.TemporaryFile;
+using Umbraco.Cms.Core.Services;
+
+namespace Umbraco.Cms.Tests.Integration.ManagementApi.TemporaryFile;
+
+[TestFixture]
+public class DeleteTemporaryFileControllerTests : ManagementApiUserGroupTestBase<DeleteTemporaryFileController>
+{
+    protected override Expression<Func<DeleteTemporaryFileController, object>> MethodSelector =>
+        x => x.Delete(_key);
+
+    private ITemporaryFileService _temporaryFileService;
```

Comment:
private variable declarations and setup method should be placed at the top of the class to maintain consistency with others.